sublime_jedi/helper.py

```diff
@@ -33,28 +48,97 @@ def docstring_panel(view, docstring):
         sublime.status_message('Jedi: No results!')
 
 
-def docstring_tooltip(view, docstring, content_builder):
+def docstring_tooltip(view, docstring, location=None):
     """Show docstring in popup.
 
     :param view (sublime.View): current active view
     :param docstring (basestring): python __doc__ string
-    :param content_builder (callable): callable object should accept docstring
-        and return content read for popup (a html text)
+    :param location (int): The text point where to create the popup
     """
-    if docstring:
-        content = content_builder(docstring)
-        view.show_popup(content, max_width=512)
+    if not docstring:
+        return sublime.status_message('Jedi: No results!')
+
+    if location is None:
+        location = view.sel()[0].begin()
+
+    # fallback if mdpopups is not available
+    if not _HAVE_MDPOPUPS:
```

Comment:
The previous way was juggling duties back and forth due to having a `content_builder` callable parameter that just does something with the other argument. That seems backwards, since the function only cares about the result of that, so either only that should be passed to the function or the logic to determine `content_builder` be contained in the function itself (as it is now).